tracer/build/_build/Build.Shared.Steps.cs

```diff
@@ -278,12 +322,6 @@ partial class Build
             var (arch, extension) = GetUnixArchitectureAndExtension();
             var dest = MonitoringHomeDirectory / arch / $"{NativeTracerProject.Name}.{extension}";
 
-            // The profiler has a different minimum glibc version to the tracer.
-            // The _overall_ minimum is the highest of the two, but as we don't
-            // currently enable the profiler on ARM64, we take the .NET runtime's minimum
-            // glibc as our actual minimum in practice. Before we can enable the profiler
-            // on arm64 we must first ensure we bring this glibc version down to 2.23.
-            //
```

Comment:
Similar to elsewhere, I think we actually need to _expand_ this so that we check the glibc requirements for all the libraries we load, including libdatadog (and potentially libddwaf, though I think that is compiled like the native loader?)

Reply:
👍